arelle/plugin/validate/EFM/config.xml

```diff
@@ -120,8 +120,6 @@
       deiDocumentPeriodEndDateElement="DocumentPeriodEndDate"
       deiFilerIdentifierElement="EntityCentralIndexKey"
       deiFilerNameElement="EntityRegistrantName"
-      logLevelFilter="(?!.*-semantic$)"
-      logCodeFilter="(?!EFM.6.03.03.matchInstance|EFM.6.05.28.linkrole|EFM.6.07.06|EFM.6.10.01.missingLabelLinkbase)"

```

Comment:
Why were these removed? I would've expected log/level filters to align with `efm-pragmatic` not `efm-strict`